lib/ood_core/job/adapters/kubernetes/helper.rb

```diff
@@ -31,7 +31,7 @@ def info_from_json(pod_json: nil, service_json: nil, secret_json: nil, ns_prefix
 
     pod_hash.deep_merge!(service_hash)
     pod_hash.deep_merge!(secret_hash)
-    OodCore::Job::Adapters::Kubernetes::K8sJobInfo.new(pod_hash)
+    OodCore::Job::Adapters::Kubernetes::K8sJobInfo.new(ood_connection_info: pod_hash[:ood_connection_info], **pod_hash)
```

Comment:
Does just `**pod_hash` work here? I feel like it should.

Reply:
Or actually you changed the interface to so `**pod_hash` should _definitely_ work.

Reply:
Yep, it does. THis change was left over when trying to find clean way to handle that method.  I pushed commit to use just `**pod_hash`

Reply:
For some reason the commit shows up on Github but I don't see the normal "outdated" header on this thread.  When I view changes too it still shows this diff without changes in e39a9c67bfe0c3a88b68c78e676cb77309a00c2f